logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/ConversationRepository.kt

```diff
@@ -566,6 +567,15 @@ internal class ConversationDataSource internal constructor(
             .map { it.map { conversationIdEntity -> idMapper.fromDaoModel(conversationIdEntity) } }
     }
 
+    override suspend fun insertConversationFromMigration(conversations: List<Conversation>): Either<CoreFailure, Unit> {
```

Comment:
I think we don't have to mention that it's used for the migration, maybe we'll find some other cases in the future 😄 also, the name is singular "insertConversation" but the param is actually a list 😉